extism.go

```diff
@@ -194,15 +194,15 @@ func (plugin Plugin) SetConfig(data map[string][]byte) error {
 	return nil
 }
 
-/// FunctionExists returns true when the name function is present in the plugin
+// / FunctionExists returns true when the name function is present in the plugin
```

Comment:
Is this extra slash a mistake? it kind of looks like I may have used Rust's doc comments when writing comments for Go and the formatter didn't like it so it changed it to this.

Reply:
Interesting. maybe my editor did that